bundles/org.openhab.binding.venstarthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -89,6 +146,141 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="fanMode">
+		<item-type>String</item-type>
+		<label>Fan Mode</label>
+		<description>Current Fan Mode</description>
+		<state readOnly="false">
+			<options>
+				<option value="auto">Auto</option>
+				<option value="on">On</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fanModeRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Fan Mode (Raw)</label>
+		<description>Current Fan Mode, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fanState">
+		<item-type>String</item-type>
+		<label>Fan State</label>
+		<description>Current Fan State</description>
+		<state readOnly="true">
+			<options>
+				<option value="off">Off</option>
+				<option value="on">On</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fanStateRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Fan State (Raw)</label>
+		<description>Current Fan State, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="scheduleMode">
+		<item-type>String</item-type>
+		<label>Schedule Mode</label>
+		<description>Current Schedule Mode</description>
+		<state readOnly="false">
+			<options>
+				<option value="disabled">Disabled</option>
+				<option value="enabled">Enabled</option>
+			</options>
+		</state>
+	</channel-type>
```

Comment:
Why not rather use a switch type ?
Just a comment, not a request for change.

Reply:
see previous comment

Reply:
@digitaldan : is it ok for you like that ? I believe it should rather be a switch.

Reply:
This would be better as a switch i think , but i'm ok either way.  Thermostats can be a little weird with toggles, FanMode for example is ON/Auto, but not off, so a switch here would actually be confusing as there's nothing to turn off (the fan always has to run if heating or cooling, but can be optionally set to run independently )

Reply:
So you can ignore my comment for this one.